apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -811,6 +815,41 @@ export class InstallManager {
     return shrinkwrapIsUpToDate;
   }
 
+  private _normalizeChmodFieldInTarHeader(tarballFile: string): void {
+    const buffer: Buffer = FileSystem.readFileToBuffer(tarballFile);
```

Comment:
Can we use a mechanism that doesn't read the entire file into memory? It may have utility in other places, even if we know that all the files we will be manipulating here are small.

Reply:
Resolving this as we now no longer read the file